docs/using_python_client_with_hazelcast.rst

```diff
@@ -378,6 +378,76 @@ A Reliable Topic usage example is shown below.
     # Publish a message to the Topic
     topic.publish("Hello to distributed world")
 
+Using ReliableMessageListener
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`ReliableMessageListener
+<https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/topic/ReliableMessageListener.java>`__ is a `MessageListener
+<https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/topic/MessageListener.java>`__ to better integrate with the reliable topic.
+
+If a regular MessageListener is registered on a reliable topic, the message listener works fine, but it can't do much more than listen to messages.
+If a ReliableMessageListener is registered on a normal topic, only the MessageListener methods are called.
+
+The following is an example Reliable Message Listener class.
+
+.. code:: python
+
+    class MyListener(ReliableMessageListener):
+        def on_message(self, message):
+            print("Received new message: ", message)
+
+        def retrieve_initial_sequence(self):
+            print("Listener function retrieve_initial_sequence is called")
+            return 0
+
+        def store_sequence(self, sequence):
+            print("Listener function store_sequence is called with sequence: ", sequence)
+            pass
+
+        def is_loss_tolerant(self):
+            print("Listener function is_loss_tolerant is called")
+            return True
+
+        def is_terminal(self, error):
+            print("Listener function is_terminal is called with error: ", error)
+            return False
+
+        def on_cancel(self):
+            print("Listener function on_cancel is called")
+
+Durable Subscription
```

Comment:
Btw, I think it might make sense to move these detailed information to the ReliableMessageListener's documentation (docstring comment) and give a link to it here.